notes/style-guide.md

```diff
@@ -4,31 +4,44 @@ Style guide for the standard library
 This is very much a work-in-progress and is not exhaustive.
 
 ## File structure
+Agda programs are structured in modules. The first module in each file of the library is the top-level module whose name always matches the filename.
 
-#### Module imports
+#### Module import guidelines
 
 * All module imports should be placed at the top of the file immediately
   after the module declaration.
 
-* If the module takes parameters that require imports from other files
+* If the module takes parameters that require imports from other files,
   then those imports only may be placed above the module declaration.
 
 * If it is important that certain names only come into scope later in
   the file then the module should still be imported at the top of the
-  file but it can be given a shorter name using `as` and then opened
-  later on in the file when needed, e.g.
+  file but it can be given a shorter name using the keyword `as` and then opened later on in the file when needed, e.g.
   ```agda
   import Data.List.Relation.Binary.Equality.Setoid as SetoidEquality
   ...
   ...
   open SetoidEquality S
   ```
 
-* The list of module imports should be in alphabetical order.
-
-* When using only a few items from a module, the items should be
-  enumerated in the import with `using` in order to make dependencies
-  clearer.
+* Changing the naming of an imported item is performed with the `renaming` directive, e.g.
```

Comment:
Again this seems like a general statement about Agda rather than about style.

Reply:
I saw that the `renaming` directive is also used in the stdlib and i thought it would be good to have a starting point in the guide to make the use explicit and under which conditions.

Reply:
I see your point, but I also think it's better to leave such explanations to the official Agda documentation which will always do a much better and more thorough job of it. As with below, maybe you could add a link to the "ReadTheDocs" page instead? (just make sure to link `latest` rather than `2.6.0` :smile:)